microprofile/messaging/core/src/main/java/io/helidon/microprofile/messaging/AbstractMessagingMethod.java

```diff
@@ -72,18 +79,23 @@ void validate() {
     }
 
     void init(BeanManager beanManager, Config config) {
+        this.config = config;
         this.beanInstance = ChannelRouter.lookup(bean, beanManager);
     }
 
-    Method getMethod() {
+    public String getName() {
```

Comment:
Why are these methods changed to `public`? Also if this is expected to be exposed by a `public` subclass, please add javadocs.